File: Search/Linear_Search/linear_search.py
def linear_search(arr, x):
    for i in range(len(arr)):
        if arr[i] == x:
            return i
    return -1

if __name__ == "__main__":
    data = [10, 23, 45, 70, 11, 15]
    target = 70
    result = linear_search(data, target)
    print(f"Element found at index: {result}" if result != -1 else "Element not found")

File: Search/Linear_Search/linear_search.cpp
#include <iostream>
#include <vector>

int linearSearch(const std::vector<int>& arr, int x) {
    for (int i = 0; i < arr.size(); i++) {
        if (arr[i] == x) return i;
    }
    return -1;
}

int main() {
    std::vector<int> data = {10, 23, 45, 70, 11, 15};
    int x = 70;
    int result = linearSearch(data, x);
    if (result != -1) std::cout << "Element found at index " << result;
    else std::cout << "Element not found";
    return 0;
}
